apps/dot-voting/app/App.js

```diff
@@ -65,51 +63,63 @@ class App extends React.Component {
     this.setState({ panelActive: false })
   }
 
+  handleResolveLocalIdentity = address => {
+    return this.props.api.resolveAddressIdentity(address).toPromise()
+  }
+
+  handleShowLocalIdentityModal = address => {
+    return this.props.api
+      .requestAddressIdentityModification(address)
+      .toPromise()
```

Comment:
Do we have a ticket for moving all of this to the `IdentityProvider` component itself? It seems we want to do this for every app that uses it.